CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# dbt_stripe_source v0.7.4
+## Under the Hood
+- The `plan_interval` field within the `stg_stripe__plan` model has been aliased within the `get_plan_columns()` macro to allow for more flexibility across warehouses. ([#58](https://github.com/fivetran/dbt_stripe_source/pull/58))
+- BuildKite testing has been added. ([#58](https://github.com/fivetran/dbt_stripe_source/pull/58))
```

Comment:
This being the BuildKite testing? I don't think it is to necessary knowing that it will be an additional item. I just added it here since there were a number of files added outside of the migration updates and wanted to call attention to that.